public/pages/DetectorResults/containers/AnomalyResultsTable.tsx

```diff
@@ -29,12 +28,18 @@ import { AnomalyData } from '../../../models/interfaces';
 import { getTitleWithCount } from '../../../utils/utils';
 import { convertToCategoryFieldAndEntityString } from '../../utils/anomalyResultUtils';
 import { HeatmapCell } from '../../AnomalyCharts/containers/AnomalyHeatmapChart';
+import { getSavedObjectsClient, getNotifications, getDataSourceEnabled } from '../../../services';
+
+//@ts-ignore
+const EuiBasicTable = EuiBasicTableComponent as any;
 
 interface AnomalyResultsTableProps {
   anomalies: AnomalyData[];
   isHCDetector?: boolean;
   isHistorical?: boolean;
   selectedHeatmapCell?: HeatmapCell | undefined;
+  detectorIndex: string[];
```

Comment:
nitpick: should this be like detectorIndices? to match other fields

could potentially utilize the `dataset` type or `data structure` type https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/data/common/datasets/types.ts#L40 so that instead of passing two new props you just pass a dataset that has insight on the underlying indices and the timefieldName

Reply:
updated the prop name s `detectorIndices`